src/locales/de.json

```diff
@@ -816,6 +826,9 @@
         },
         "WebcamPanel": {
             "All": "Alle",
+            "ConnectingTo": "Verbinde zu {url}",
+            "Disconnected": "Getrennt",
+            "ErrorWhileConnecting": "Fehler während dem verbinden zu {url}",
```

Comment:
Correct would be:
`Fehler während dem Verbinden zu {url}`

Reply:
To make it clear. It's a capitalized `V` as `dem` indicates a noun and therefore we have to capitalize it